src/channel.ts

```diff
@@ -1555,4 +1573,11 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
     this.disconnected = true;
     this.state.setIsUpToDate(false);
   }
+
+  _flushWsEventQueue() {
+    while (this.wsEventQueue.length) {
+      const event = this.wsEventQueue.pop();
```

Comment:
I think FIFO would make more sense than LIFO.

Reply:
`unshift` is expensive compared to `pop`, but I can't imagine this queue being huge.